litmus-portal/frontend/src/views/WorkflowDetails/WorkflowNodeInfo/styles.ts

```diff
@@ -45,6 +45,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     margin: theme.spacing(1, 0),
   },
 
+  buttonAlign: {
+    paddingLeft: theme.spacing(0),
```

Comment:
I guess this is equivalent to no padding. This entire style class can be removed.